### PR TITLE
Develop plugin handling + webui fixes

### PR DESCRIPTION
**Pull request by bendikro**
This commit relies in the fix in https://github.com/deluge-torrent/deluge/pull/65

### CODE REVIEW

File: deluge/ui/web/server.py [hunk 1]
```diff
@@ -577,12 +577,13 @@ def start(self, standalone=True):
...
-        Returns:
-            Deferred
         """
-        log.info("%s %s.", _("Starting server in PID"), os.getpid())
+        if self.socket:
+            log.warn("DelugeWeb is already running and cannot be started")
+            return
+
         self.standalone = standalone
+        log.info("%s %s.", _("Starting server in PID"), os.getpid())
```
Comment (cas--) [inline deluge/ui/web/server.py]: I think remove translation hint and reword: "Starting webui server at PID %s"

Comment (bendikro) [inline deluge/ui/web/server.py]: Sure

**Comment by bendikro** [comment]
Updated to avoid stack trace when killing deluge-web with CTRL-C

**Comment by cas--** [comment]
could you rebase plese

**Comment by bendikro** [comment]
Oops, sorry

**Comment by bendikro** [comment]
Fixed

**Comment by cas--** [comment]
Merged: [70d8b65f0a499]

**Pull request closed without merging**